### PR TITLE
[CU-2my8v57] chore(.config): spellcheck CI dictionary

### PR DESCRIPTION
**Pull request by Martin-Kiss**
## Issue
https://app.clickup.com/t/2my8v57

## Description
Extend CI spellcheck dictionary
Draft to implement spellchecking according to #682  

## Checklist

~~- [ ] I have updated the cargo docs to reflect changes made by this PR _(if applicable)_~~
~~- [ ] I have updated the `book/` to reflect changes made by this PR _(if applicable)_~~

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2my8v57 Extending spellcheck CI dictionary](https://app.clickup.com/t/2my8v57)

**Review by PoisonPhang — APPROVED** [review]

**Review by Dom-Roth — APPROVED** [review]

File: .config/spellcheck.dic [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+backfilling
+benluelo
+bignumbers
+blocknumber
+blockweight
+bonder
+bootnodes
+builtins
+CLI
+CLI
```
Comment (vivekvpandya) [inline .config/spellcheck.dic:23]: dup

File: .config/spellcheck.dic [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+DeFi
+deps
+devel
+devnet
+Devnets
+Dex
+DEX
+dispatchable
+Dispatchable
+Dispatchable
```
Comment (vivekvpandya) [inline .config/spellcheck.dic:52]: dup

**Review by benluelo — APPROVED** [review]
Some duplicate lines in the dictionary but lgtm otherwise

**Review by cor — APPROVED** [review]
LGTM

**Review by KaiserKarel — APPROVED** [review]

**Review by haroldsphinx — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Pull request merged**